### DISCUSSION TITLE
[FEATURE]需要PHP8.1支持更多扩展，以符合应用需要

### DISCUSSION THREAD

**Discussion by panluoyant** [1Panel-dev/1Panel#6088] [Ideas]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

我是用的一些wiki知识库（例如Xwiki，MWiki）程序需要一下扩展，希望可以添加
![image](https://github.com/user-attachments/assets/dd9f8e18-72a5-4b6f-9427-45cc8d830b0d)

**Comment by wanghe-fit2cloud** [comment]
1、列表中不存在的扩展，可以手动输入之后选择，例:输入 sockets ，然后在下拉列表中选择第一个，
[查看扩展列表](https://1panel.cn/docs/user_manual/websites/php/#php_1)
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/75c55a5a-d1c8-4d96-ac23-171d5b0389e5">

2、自定义扩展使用教程（以 swoole_loader 为例） 
https://bbs.fit2cloud.com/t/topic/2073
